ludwig/encoders/text_encoders.py

```diff
@@ -2180,11 +2183,15 @@ def _maybe_resize_token_embeddings(self, transformer, vocab_size: Optional[int]
     def forward(self, inputs: torch.Tensor, mask: Optional[torch.Tensor] = None):
         if mask is not None:
             mask = mask.to(torch.int32)
-        transformer_outputs = self.transformer.module(
+
+        kwargs = dict(
```

Comment:
Would it make sense to add any aux checks for incompatible config parameters passed to specific encoders?

Reply:
Hmm, not sure what you had in mind here. The issue is predominately that there are many different HF model implementations that all take different params. So we need to make sure we don't accidentally pass in the wrong params.

Reply:
Oh, apologies, on followup look this seems like something that would be difficult to catch in the schema. I was misremembering how we used `AutoTransformer`.